pkg/diskrepart/disk.go

```diff
@@ -455,7 +455,7 @@ func (dev Disk) expandFilesystem(device string) (err error) {
 		if err != nil {
 			return err
 		}
-		err = dev.sys.Mounter().Mount(device, tmpDir, "auto", []string{})
+		err = dev.sys.Mounter().Mount(device, tmpDir, "", []string{})
```

Comment:
Just changed it because according to the mount implementation this is how it should be https://github.com/kubernetes/mount-utils/blob/71d03c6f5d1869ee95fdeca1000d270370345125/mount_linux.go#L158-L162